src/sim65/6502.c

```diff
@@ -215,7 +215,9 @@ int PrintCycles;
     ZPAddr = MemReadByte (Regs.PC+1);                           \
     Addr = MemReadZPWord (ZPAddr);                              \
     if (PAGE_CROSS (Addr, Regs.YR))                             \
+    {                                                           \
```

Comment:
Really sorry to bother you but the open curly bracket is supposed to be at the end of the previous line.